### DISCUSSION TITLE
Probleme beim Abfahren -CP : Ziel ist nicht auf einem Feld. ---CP: Target is not on a field.

### DISCUSSION THREAD

**Discussion by Toaogurch** [Courseplay/Courseplay_FS22#3190] [General]
The driver generates the error
I've now tried this on field 64! The same problem, thresher and downdriver have the same target field 64. When I set the downdriver I get the error, CP: Target is not on a field.
Both vehicles are on field 64????

What would I give if I can get rid of this...!!!!

Thank you very much for your effort in advance!

Please help me!!!!
----------------------------------------------------------------
Der Abfahrer erzeugt den Fehler
Ich habe das jetzt auf dem Feld 64 ausprobiert! Das gleiche Problem, Drescher und Abfahrer haben das gleiche Ziel Feld 64. Wenn ich den Abfahrer einstelle bekomme ich den Fehler, CP : Ziel ist nicht auf einem Feld.
Beide Fahrzeuge stehen auf dem Feld 64????

Was gebe ich dafür wenn ich das beseitigen kann...!!!!

Herzlichen Dank für Eure Mühe im Voraus!

Bitte helft mir!!!!

**Comment by Tensuko** [comment]
Das klingt mir sehr nach Anleitung nicht gelesen.
Welches Ziel vom Drescher ist auf dem Feld, das "Ziel" vom Drescher intressiert CP nicht.
Es muss lediglich das Feld mit dem Marker markiert werden.

Wie gesagt, Anleitung lesen (ingame oder unter courseplay.dev) da steht alles beschrieben!

**Comment by Toaogurch** [comment]
Hallo Tensuko,

Danke für den Hinweis!!! Ich dachte das kann ja nicht so schwer sein!
So kann man sich irren…!!!!
Wenn ich doch noch Fragen habe stelle ich die am Besten hier!!!!
Ihr habt ein sehr schönes, sehr gutes Programm entwickelt!!!!

Herzlichen Dank dafür!!!!

**Comment by Toaogurch** [comment]
Hallo Tensuko,
es tut mir sehr leid aber da bin ich wieder. Ich bin an Alzheimer erkrankt und habe Schwierigkeiten mich an verschiedene Sachen zu erinnern! Damit ich aber geistig fit bleibe deshalb spiele ich LS22!
Der Fehler ist immer noch da. Ich habe das Spiel gelöscht und neu angefangen. Jetzt funktioniert das abfahren auf dem Feld dafür funktioniert es auf anderen Feldern nicht! 
Ich verstehe das nicht...! 
Kannst Du mir helfen...?

**Reply by Toaogurch** [reply]
Auf anderen Feldern klappt es super mit genau den Einstellungen die ich beschrieben habe...!?!?!?!

**Reply by Tensuko** [reply]
Wenn du im HUD das Icon anklickst, landest du ja auf der Helfer Karte. Auf der rechten Seite gibt es dann "knöpfe" mit koordinaten, das sind die Ziele die du auf dein neues Feld setzen musst, damit der Abfahrer auch auf dem richtigen Feld nach dem Drescher sucht.
Das Feld wird dann weis umrandet.
